docs/source/usage.rst

```diff
@@ -880,6 +833,53 @@ Creates a copy of a DictConfig that contains only specific keys.
       b: 10
     <BLANKLINE>
 
+OmegaConf.is_missing
+^^^^^^^^^^^^^^^^^^^^
+
+Tests if a value is missing ('???').
+
+.. doctest::
+
+    >>> cfg = OmegaConf.create({
+    ...         "foo" : 10, 
+    ...         "bar": "???"
+    ...     })
+    >>> assert not OmegaConf.is_missing(cfg, "foo")
+    >>> assert OmegaConf.is_missing(cfg, "bar")
+
+OmegaConf.is_interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tests if a value is an interpolation.
+
+.. doctest::
+
+    >>> cfg = OmegaConf.create({
+    ...         "foo" : 10, 
+    ...         "bar": "${foo}"
+    ...     })
+    >>> assert not OmegaConf.is_interpolation(cfg, "foo")
+    >>> assert OmegaConf.is_interpolation(cfg, "bar")
+
+
+OmegaConf.{is_config, is_dict, is_list}
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tests if an object is an OmegaConf object, or if it's representing a list or a dict.
+
+.. doctest::
+
+    >>> # dict:
+    >>> d = OmegaConf.create({"foo": "bar"})
+    >>> assert OmegaConf.is_config(d)
+    >>> assert OmegaConf.is_dict(d)
+    >>> assert not OmegaConf.is_list(d)
+    >>> # list:
+    >>> l = OmegaConf.create([1,2,3])
+    >>> assert OmegaConf.is_config(l)
+    >>> assert OmegaConf.is_list(l)
+    >>> assert not OmegaConf.is_dict(l)
+
 Debugger integration
 ^^^^^^^^^^^^^^^^^^^^
```

Comment:
While you are at it, can you make debugger integration into a top level item? right now it's under misc functions.
Otherwise looks good.

Reply:
Done in https://github.com/omry/omegaconf/pull/614/commits/5c64c828def8d98fa3d6d6a6b170e1f278c57e1d

Reply:
All good, thanks!